CHANGES.rst

```diff
@@ -25,6 +25,10 @@ Deprecated
 Features added
 --------------
 
+* #12975: Add the possibility to avoid rendering a trailing comma in Python and
+  Javascript multi-line signatures, via
+  :confval:`python_trailing_comma_in_multi_line_signatures` and
+  :confval:`javascript_trailing_comma_in_multi_line_signatures`, respectively.
```

Comment:
```suggestion
* #12975: Add the possibility to avoid rendering a trailing comma in Python and
  Javascript multi-line signatures via
  :confval:`python_trailing_comma_in_multi_line_signatures` and
  :confval:`javascript_trailing_comma_in_multi_line_signatures` respectively.
```